<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaSettingsGroupBox</class>
 <widget class="QGroupBox" name="MediaSettingsGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="title">
   <string>Media Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>File Patch:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileNameLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string>Error Label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="mediaSettinsSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="previewGroupBox">
      <property name="title">
       <string>Preview:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Phonon::VideoPlayer" name="previerVideoPlayer" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="Phonon::SeekSlider" name="previewSeekSlider" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="iconVisible" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="currentTimeLabel">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="playHorizontalLayout">
         <item>
          <widget class="QToolButton" name="playPauseToolButton">
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/control.png</normaloff>:/icons/control.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="stopToolButton">
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/control-stop-square.png</normaloff>:/icons/control-stop-square.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="previewMuteToolButton">
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/speaker-volume.png</normaloff>:/icons/speaker-volume.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Phonon::VolumeSlider" name="previewVolumeSlider" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="muteVisible" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="mediaTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Media Config</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="titleLabel">
                <property name="text">
                 <string>Title:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="titleLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="3">
               <widget class="QToolButton" name="currentTimeStartToolButton">
                <property name="icon">
                 <iconset resource="icons.qrc">
                  <normaloff>:/icons/film-timeline.png</normaloff>:/icons/film-timeline.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QToolButton" name="currentTimeEndToolButton">
                <property name="icon">
                 <iconset resource="icons.qrc">
                  <normaloff>:/icons/film-timeline.png</normaloff>:/icons/film-timeline.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QTimeEdit" name="startPositionTimetimeEdit">
                <property name="displayFormat">
                 <string>H:mm:ss:zzz</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>End Positon:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Start Position:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QTimeEdit" name="endPositionTimeTimeEdit">
                <property name="displayFormat">
                 <string>H:mm:ss:zzz</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_7">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/brightness.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Brightness:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_8">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/contrast.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_9">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/color.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_4">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/color-swatch.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Contrast:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Saturation:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Hue:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QDoubleSpinBox" name="brightnessDoubleSpinBox">
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QDoubleSpinBox" name="contrastDoubleSpinBox">
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QDoubleSpinBox" name="saturationDoubleSpinBox">
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QDoubleSpinBox" name="hueDoubleSpinBox">
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_12">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/ui-slider.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_13">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/ui-slider-100.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="startDateImage">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/calendar-next.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLabel" name="stardDateLabel">
                <property name="text">
                 <string>Start Date:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QDateEdit" name="stardDateDateEdit">
                <property name="currentSection">
                 <enum>QDateTimeEdit::DaySection</enum>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLabel" name="endDateImage">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/calendar-previous.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLabel" name="endDateLabel">
                <property name="text">
                 <string>End Date:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="6">
               <widget class="QDateEdit" name="endDateDateEdit">
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QLabel" name="startTimeImage">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/clock-select.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="2" column="5">
               <widget class="QLabel" name="startTimeLabel">
                <property name="text">
                 <string>Start Time:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="6">
               <widget class="QTimeEdit" name="startTimeTimeEdit">
                <property name="displayFormat">
                 <string>hh:mm</string>
                </property>
               </widget>
              </item>
              <item row="3" column="4">
               <widget class="QLabel" name="endTimeImage">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/clock-select-remain.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="3" column="5">
               <widget class="QLabel" name="endTimeLabel">
                <property name="text">
                 <string>End Time:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="6">
               <widget class="QTimeEdit" name="endTimeTimeEdit">
                <property name="displayFormat">
                 <string>hh:mm</string>
                </property>
               </widget>
              </item>
              <item row="4" column="4">
               <widget class="QLabel" name="label_14">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/notification-counter.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="4" column="5">
               <widget class="QLabel" name="numberToShowsLabel">
                <property name="text">
                 <string>Number To Shows:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="6">
               <widget class="QSpinBox" name="numberToShowsSpinBox">
                <property name="specialValueText">
                 <string>∞</string>
                </property>
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>-1</number>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
                <property name="value">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item row="5" column="4">
               <widget class="QLabel" name="label_15">
                <property name="pixmap">
                 <pixmap resource="icons.qrc">:/icons/weight.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="5" column="5">
               <widget class="QLabel" name="randomWeightLabel">
                <property name="text">
                 <string>Random Weight:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="6">
               <widget class="QSpinBox" name="randomWeightSpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Media Info</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="mediaInfoTableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
